forc-plugins/forc-node/src/consts.rs

```diff
@@ -0,0 +1,17 @@
+/// Minimum fuel-core version supported.
+pub const MIN_FUEL_CORE_VERSION: &str = "0.40.0";
+
+pub const TESTNET_RESERVED_NODE: &str = "/dns4/p2p-testnet.fuel.network/tcp/30333/p2p/16Uiu2HAmDxoChB7AheKNvCVpD4PHJwuDGn8rifMBEHmEynGHvHrf";
+pub const TESTNET_SERVICE_NAME: &str = "fuel-sepolia-testnet-node";
+pub const TESTNET_SYNC_HEADER_BATCH_SIZE: u32 = 100;
+pub const TESTNET_RELAYER_LISTENING_CONTRACT: &str = "0x01855B78C1f8868DE70e84507ec735983bf262dA";
+pub const TESTNET_RELAYER_DA_DEPLOY_HEIGHT: u32 = 5827607;
+pub const TESTNET_RELAYER_LOG_PAGE_SIZE: u32 = 500;
+pub const TESTNET_SYNC_BLOCK_STREAM_BUFFER_SIZE: u32 = 30;
+
+pub const MAINNET_BOOTSTRAP_NODE: &str = "/dnsaddr/mainnet.fuel.network";
```

Comment:
We should add a `.` to the end of this hostname to avoid any potential DNS resolution issues on the client side. So the value should be `/dnsaddr/mainnet.fuel.network.`